### ISSUE TITLE
TemporalFrame doesn't work with HighLevelWCSWrapper

### ISSUE THREAD

**Issue by Cadair** [spacetelescope/gwcs#252]
Despite the refactoring of how we handle `TemporalFrame` having a relative frame (model returns delta T in seconds) still doesn't seem to work correctly with `HighLevelWCSWrapper.pixel_to_world`.

**Comment by nden** [comment]
Please post an example.

**Comment by Cadair** [comment]
Will do, just as soon as break it down to a MWE :smile:

**Comment by Cadair** [comment]
here we go:
```python
import astropy.units as u
from astropy.modeling import models
from astropy.time import Time
from astropy.wcs.wcsapi import HighLevelWCSWrapper

import gwcs
import gwcs.coordinate_frames as cf

transform = models.Linear1D(10*u.s/u.pix, 0*u.s)

input_frame = cf.CoordinateFrame(1, "pixel", (0,), unit=u.pix)
output_frame = cf.TemporalFrame(Time("2010-01-01T00:00"), unit=u.s, axes_order=(0,))

w = gwcs.WCS(forward_transform=transform, output_frame=output_frame,
             input_frame=input_frame)
w.pixel_to_world(10)
```
```
<Time object: scale='utc' format='isot' value=2010-01-01T00:01:40.000>
```
```python
w2 = HighLevelWCSWrapper(w)
w2.pixel_to_world(10)
```
```
~/DKIST/test_gwcs_time_wrapper.py in <module>
     16 
     17 w2 = HighLevelWCSWrapper(w)
---> 18 w2.pixel_to_world(10)

~/Git/astropy/astropy/wcs/wcsapi/high_level_api.py in pixel_to_world(self, *pixel_arrays)
    225         for key in default_order(components):
    226             klass, ar, kw = classes[key]
--> 227             result.append(klass(*args[key], *ar, **kwargs[key], **kw))
    228 
    229         if len(result) == 1:

~/Git/astropy/astropy/time/core.py in __init__(self, val, val2, format, scale, precision, in_subfmt, out_subfmt, location, copy)
    396         else:
    397             self._init_from_vals(val, val2, format, scale, copy,
--> 398                                  precision, in_subfmt, out_subfmt)
    399             self.SCALES = TIME_TYPES[self.scale]
    400 

~/Git/astropy/astropy/time/core.py in _init_from_vals(self, val, val2, format, scale, copy, precision, in_subfmt, out_subfmt)
    451         # Parse / convert input values into internal jd1, jd2 based on format
    452         self._time = self._get_time_fmt(val, val2, format, scale,
--> 453                                         precision, in_subfmt, out_subfmt)
    454         self._format = self._time.name
    455 

~/Git/astropy/astropy/time/core.py in _get_time_fmt(self, val, val2, format, scale, precision, in_subfmt, out_subfmt)
    502                 pass
    503         else:
--> 504             raise ValueError(f'Input values did not match {err_msg}')
    505 
    506     @classmethod

ValueError: Input values did not match the format class isot
```

**Comment by Cadair** [comment]
I opened #253 which "fixes" this

**Comment by nden** [comment]
Fixed in #253
